hopper/test_flash_attn.py

```diff
@@ -49,6 +51,18 @@
 )
 
 
+def run_fake_check(fn):
+    def wrapper(*args, **kwargs):
+        fake_check(fn, args, kwargs)
+        return fn(*args, **kwargs)
+    return wrapper
+
```

Comment:
Do we have a sense of how comprehensive the existing tests in this file are? Are they good at exercising a variety of inputs?